lib/wanda/catalog.ex

```diff
@@ -69,4 +72,26 @@ defmodule Wanda.Catalog do
       argument: argument
     }
   end
+
+  defp map_values(%{"values" => values}) do
+    Enum.map(values, &map_value/1)
+  end
+
+  defp map_values(_), do: []
+
+  defp map_value(%{"name" => name, "default" => default} = value) do
+    conditions = Map.get(value, "conditions", [])
```

Comment:
```elixir
value
|> Map.get("conditions", [])
|> Enum.map(fn %{"value" => value, "when" => expression} ->
        %Condition{
          value: value,
          expression: expression
        }
      end)
```

And then you can actually build the Value as

```elixir
%Value{name: name, default: default, conditions: conditions}
```

Reply:
Done